drucker/drucker_dashboard_servicer.py

```diff
@@ -175,23 +175,49 @@ def EvaluateModel(self,
         """ Evaluate your ML model and save result.
         """
         first_req = next(request_iterator)
-        save_path = first_req.data_path
-        if not self.is_valid_upload_filename(save_path):
-            raise Exception(f'Error: Invalid evaluation file path specified -> {save_path}')
+        data_path = first_req.data_path
+        result_path = first_req.result_path
```

Comment:
If the file already exists, we don't need to evaluate it again. Could you load the `result_path` and return it?

Reply:
Could you add `overwrite` option?
If `overwrite` is true, we evaluate it and save it again.

Reply:
@keigohtr 
right... but actually data_path and result_path are generated based on datetime in dashboard,
so should I check the EvaluationResult existence and have `overwrite` option in dashboard API?
In dashboard, EvaluationResult is managed by ID, so it is easier to check.

Reply:
I see.

> so should I check the EvaluationResult existence and have overwrite option in dashboard API? In dashboard, EvaluationResult is managed by ID, so it is easier to check.

I agree with you. How about this?
1. calculate the hash to the data on dashboard. (dashboard)
1. send the hash also when sending a data. (dashboard)
1. check the hash code on DB (we need to store hash code at the same time). (Rekcurd)
1. ignore or store. (Rekcurd)

Reply:
@keigohtr 
I added duplication check for EvaluationResult in dashboard PR: https://github.com/rekcurd/drucker-dashboard/pull/50/commits/278ebae28db2dbef40c4f688a352bcd19f72cbb2